--- plucker_desktop/Makefile.in~	2003-10-06 23:57:17.000000000 +0200
+++ plucker_desktop/Makefile.in	2004-06-05 16:14:25.000000000 +0200
@@ -35,16 +35,16 @@
 
 TARGET          = plucker-desktop
 
-CXXFLAGS        = @CXXFLAGS@ `wx-config --cflags`
+CXXFLAGS        = @CXXFLAGS@ `wxgtk2-2.4-config --cflags`
 
-WX_LIBRARY_BASENAME	= `wx-config --basename`
-WX_RELEASE	= `wx-config --release`
+WX_LIBRARY_BASENAME	= `wxgtk2-2.4-config --basename`
+WX_RELEASE	= `wxgtk2-2.4-config --release`
 
 # Use with/without the stc depending on whether want to use styled text control ('stc')
 # Note the order of libs is vital. Since the xrc and stc libs depend on the main wx library
 # the xrc and stc have to come first. If you switch them around, you will get weird 
 # reference errors.
-LDLIBS          =-l$(WX_LIBRARY_BASENAME)_xrc-$(WX_RELEASE) -l$(WX_LIBRARY_BASENAME)_stc-$(WX_RELEASE) `wx-config --libs` 
+LDLIBS          =-l$(WX_LIBRARY_BASENAME)_xrc-$(WX_RELEASE) -l$(WX_LIBRARY_BASENAME)_stc-$(WX_RELEASE) `wxgtk2-2.4-config --libs` 
    
 CXX             = @CXX@ $(CXXFLAGS)
 CTAGS           = @CTAGS@
