diff -urN plucker_2003-09-14_12h01m33s/unix/install-plucker plucker_patched/unix/install-plucker
--- plucker_2003-09-14_12h01m33s/unix/install-plucker	2003-08-03 16:07:02.000000000 +0200
+++ plucker_patched/unix/install-plucker	2003-09-14 23:13:26.000000000 +0200
@@ -18,7 +18,7 @@
 BINPKG='@@BINPKG@@'
 
 # default install targets
-TARGET="install-prc-files install-manual install-data"
+TARGET="install-manual install-data"
 
 # set up the Python environment
 PYTHON_LIBVER = string.split(sys.version)[0]
